pmlc/target/x86/pipeline.cc

```diff
@@ -89,8 +89,8 @@ void addToPipeline(OpPassManager &pm) {
   // pm.addPass(createCanonicalizerPass());
   // pm.addPass(pmlc::dialect::pxa::createMemRefDataFlowOptPass());
   // pm.addPass(createCanonicalizerPass());
-  // pm.addPass(pmlc::dialect::pxa::createLocalizePass());
-  // pm.addPass(pmlc::dialect::pxa::createResizeTmpsPass());
+  pm.addPass(pmlc::dialect::pxa::createLocalizePass());
+  pm.addPass(pmlc::dialect::pxa::createResizeTmpsPass());
   // pm.addPass(createCanonicalizerPass());
   // pm.addPass(createCSEPass());
```

Comment:
We should probably enable these passes as well (Canonicalizer and CSE), now that Localize and ResizeTmps are running.

Reply:
Done.